### DISCUSSION TITLE
Simplera(Medtronic) companion app does not work

### DISCUSSION THREAD

**Discussion by Nzugadi** [NightscoutFoundation/xDrip#3686] [Bug reports]
We use Simplera sensor. Master not working with companion app on latest version (all permissions given). We have to use old release of Xdrip on master, so Xdrip Sync follower does not work since Google change things. 
Master phone Samsung Galaxy A10 (Android 11)

**2024/05/28 RELEASE WORKS (Event logs)**
![Event log works](https://github.com/user-attachments/assets/ea5bba63-f192-443e-9fd1-96dc006a14c0)

**2024/05/29 RELEASE DOES NOT WORK Event logs does not work****
![Event log does not work](https://github.com/user-attachments/assets/e4d6b4e2-6542-4601-ae89-8ff50adcdb10)
![Event log does not work2](https://github.com/user-attachments/assets/07f256d3-d233-4863-b408-8a1246be34d2)

¿Can anyone help us?

Thank

**Comment by jamorham** [comment]
Okay I think this is related to the changes in the target sdk. I believe there may be a workaround so I will attempt to add this to the next nightly for testing.

**Reply by Navid200** [reply]
Thanks
I didn't mean to make work for you.
I can do anything needed if it helps if you tell me.

**Comment by jamorham** [comment]
@Nzugadi please try the 27th Sept nightly and see if that resolves the issue. Thanks

**Reply by Nzugadi** [reply]
It works on master phone but not on followers 

Followers events log
![Screenshot_2024-09-27-20-25-48-47_2dd2c1bcb8f659cd696bcba2072f751b](https://github.com/user-attachments/assets/681c8b40-d1c4-432a-8edd-523f72c7266b)

**Reply by Navid200** [reply]
Is Use xDrip Cloud enabled on master and follower?

**Reply by Nzugadi** [reply]
Yes

**Reply by Navid200** [reply]
Can you show a screenshot of the status page, heading "Followers"?
Both master and follower?

**Reply by Nzugadi** [reply]
THANK YOU VERY VERY MUCH. YOU MAKE A VERY IMPORTANT LABOR.

**Reply by Navid200** [reply]
Great!
Thanks for reporting this.

@jamorham Thanks so much for fixing this

**Reply by Navid200** [reply]
To disable detail logs that I had asked you to enable, do the following.

Go to `Settings` > `Less common settings` > `Extra Logging Settings` > `Extra tags for logging`

Delete everything and hit OK.

**Comment by jamorham** [comment]
@Nzugadi Sorry I deleted your post where you showed your security key because this is a privacy and security risk if you publish it. I am glad things are working now.

**Reply by Nzugadi** [reply]
THANK YOU SO MUCH

**Comment by octaviorubio** [comment]
Hola, alguien sabe cómo configurar simplera en la aplicación xdrip? Ando perdido
Decir que tengo instalado xdrip en el móvil funcionando con mi reloj y freestyle, pero no sé cómo proceder a configurarlo con simplera

**Reply by Navid200** [reply]
@octaviorubio Simplera is a sensor.  You can set up xDrip for Simplera or freestyle.  But, not both.  
Can you explain better what you need?

**Reply by octaviorubio** [reply]
Hola navid, gracias por su rapidez en contestar.

Mi "problema" es que hasta el dia de hoy he estado usando freestyle con xdrip para volcar los datos en mi reloj. Pero he cambiado a simplera y ahora no sé cómo configurarlo para ver los datos de simplera (freestyle ya no lo uso) en el.

**Reply by Navid200** [reply]
I see.
You need to use the companion app mode: https://navid200.github.io/xDrip/docs/Follow/CompanionApp.html 

Have a look and let me know if you have any questions.

**Reply by octaviorubio** [reply]
Gracias Navid, el miércoles comienzo con simplera. En cuanto arranque me pondré a trastear con los pasos que me has facilitado 🫂
